presidio-analyzer/presidio_analyzer/predefined_recognizers/pl_pesel_recognizer.py

```diff
@@ -0,0 +1,51 @@
+from typing import List, Optional
+
+from presidio_analyzer import Pattern, PatternRecognizer
+
+
+class PlPeselRecognizer(PatternRecognizer):
+    """
+    Recognize PESEL number using regex and checksum.
+
+    For more information about PESEL: https://en.wikipedia.org/wiki/PESEL
+
+    :param patterns: List of patterns to be used by this recognizer
+    :param context: List of context words to increase confidence in detection
+    :param supported_language: Language this recognizer supports
+    :param supported_entity: The entity this recognizer can detect
+    """
+
+    PATTERNS = [
+        Pattern(
+            "PESEL",
+            r"[0-9]{2}([02468][1-9]|[13579][012])(0[1-9]|1[0-9]|2[0-9]|3[01])[0-9]{5}",
+            0.5,
```

Comment:
Can we consider reducing the default confidence to 0.4? While I don't anticipate this regex to create too many false positives, the default threshold is usually 0.4. With context words and/or checksum, this score will increase. WDYT?

Reply:
Ok